NEWS.md

```diff
@@ -17,7 +17,7 @@ Language changes
     module. For instance, `Base.r"x"` is now parsed as `Base.@r_str "x"`.
     Previously, this syntax parsed as an implicit multiplication. ([#18690])
 
-  * For every binary operator `⨳`, `a .⨳ b` is now automatically equivalent to
+  * For every binary operator `⨳` (`⨳` is only a placeholder; isn't a new operator), `a .⨳ b` is now automatically equivalent to
```

Comment:
Actually, `⨳` is an example of a real binary operator (which doesn't have a default definition).

Maybe
```
For every binary operator, such as `⨳`, .....
```